install/installer/cmd/testdata/render/minimal/output.golden

```diff
@@ -1424,8 +1424,7 @@ data:
           "gcloud": {
             "credentialsFile": "",
             "region": "",
-            "projectId": "",
-            "maximumBackupCount": 0
+            "projectId": ""
```

Comment:
Is this related to your change?

Reply:
Nope. The file was regenerated with `make generateRenderTests` following the [README](https://github.com/gitpod-io/gitpod/tree/main/install/installer/cmd/testdata/render). 

I don't know what accounts for the changes in the file besides the change to the proxy config. Maybe the file was not updated by a previous PR?

Reply:
I guess Self-Hosted is on the review and can clarify. Perhaps we're missing a CI check that would fail if the `output.golden` doesn't match anymore.